docs/articles/commands/processors/slash_commands/missing_commands.md

```diff
@@ -0,0 +1,18 @@
+---
+uid: articles.commands.command_processors.slash_commands.missing_commands
+title: Missing Commands
+---
+
+# Missing Commands
+
+> Help! I registered all my slash commands but they aren't showing up!
+
+When the Discord App (the client you use, not the bot) starts up, it fetches all the commands that are registered with the bot and caches them to a Discord channel. Unfortunately this cache is not properly updated in real-time, so it may take a while for the commands to show up.
+
+Your solution here is to reload your Discord App.
+
+> Help! They're still not showing up!
+
+Maybe some of your commands were invalid and the DSharpPlus.Commands library didn't register them. If you have text commands enabled, try mentioning your bot with the command name and see if it responds. If it doesn't, then the command is not registered.

```

Comment:
I don't jive with telling users /Maybe/ in light of commands not registering. It's indicative that we're not giving verbose/robust errors.

I would recommend rewording this to implore users to check the logs of the console window, since an exception is thrown for invalid commands. (Or, less than the expected command count will be registered, but generally the former).

(I've also just noticed this is recommended at the bottom, but I still don't necessarily agree with starting a sentence with "Maybe". :ioa:)

Reply:
is this comment resolved?